src/components/common/Application.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useState, type FormEvent } from "react";
+import axios from "axios";
+import Select from 'react-select';
+
+const Application = ( 
+  {name}: {name: string}
+) => {
+  const [formData, setFormData] = useState({
+    name: "",
+    email: "",
+    location: "",
+    describeYourself: "",
+    year: "",
+    background: "",
+    github: "",
+    time: "",
+    why: "",
+    skills: [{}],
+    books : [{}],
+    enrolled: false,
+    role:name
+  });
+
+  const [selectedSkills, setSelectedSkills] = useState<OptionType[]>([]);
+  const [selectedBooks, setSelectedBooks] = useState<OptionType[]>([]);
+
+  const skills = [
+    { value: 'Full-stack', label: 'Full-stack' },
+    { value: 'Front-end', label: 'Front-end' },
+    { value: 'Back-end', label: 'Back-end' },
+    { value: 'Dev ops', label: 'Dev ops' },
+    { value: 'UI/UX design', label: 'UI/UX design' },
+    { value: 'Prompt engineering', label: 'Prompt engineering' },
+    { value: "Rust", label: "Rust" },
+    { value: "Python", label: "Python" },
+    { value: "C++", label: "C++" },
+    { value: "Golang", label: "Golang" },
+    { value: "Graphic Design", label: "Graphic Design" },
+    { value: "Video Editing", label: "Video Editing" },
+    { value: "Product Management", label: "Product Management" },
+    { value: "Accounting", label: "Accounting" },
+    { value: "Law", label: "Law" },
+    { value: "Sales", label: "Sales" },
+    { value: "Business Operations", label: "Business Operations" },
+    { value: "Others", label: "Others" }
+  ]
+
+  const books = [
+    { value: "Mastering Bitcoin", label: "Mastering Bitcoin" },
+    { value: "Mastering Lightning Network", label: "Mastering Lightning Network" },
+    { value: "BPD", label: "BPD" },
+    { value: "LPD", label: "LPD" },
+    { value: "Learning Bitcoin through Command Line", label: "Learning Bitcoin through Command Line" },
+    { value: "Programming Bitcoin", label: "Programming Bitcoin" },
+    { value: "The Bitcoin Standard", label: "The Bitcoin Standard" },
+    { value: "Sovereign Individual", label: "Sovereign Individual" },
+    { value: "The Broken Money", label: "The Broken Money" },
+    { value: "The Blocksize War", label: "The Blocksize War" },
+    { value: "Others", label: "Others" }
+  ]
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const { name, value } = e.target;
+    setFormData((prevState) => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
+    formData.skills = selectedSkills;
+    formData.books = selectedBooks;
+    e.preventDefault();
+    console.log("userdata", formData);
+    axios.post("http://13.232.129.218/register", formData).then((response) => {
```

Comment:
.then is not needed here. Can we use async await and maybe show in the UI that submission is complete